src/main/kotlin/uk/gov/justice/digital/hmpps/hmppshdcapi/licences/LicenceService.kt

```diff
@@ -56,4 +64,86 @@ class LicenceService(
       addressTown = addressObject.addressTown,
       postCode = addressObject.postCode,
     )
+
+  private fun formatCurfewHoursObject(curfewHours: CurfewHours, licenceId: Long?): List<CurfewTimes> {
+    val day1 =
+      CurfewTimes(
+        licenceId = licenceId,
+        fromDay = "Monday",
+        fromTime = LocalTime.parse(curfewHours.mondayFrom),
+        untilDay = "Tuesday",
+        untilTime = LocalTime.parse(curfewHours.tuesdayUntil),
+        curfewTimesSequence = 1,
+        createdTimestamp = LocalDateTime.now(clock),
```

Comment:
**suggestion:** createdTimestamp is when the entity was created, which we don't really know in HDC and is a bit misleading to always say "now" whenever its retrieved. 

I think this is more of an implementation detail of how we will store it in CVL and doesn't need to become part of an public API interface (I don't think it's used in any frontend), so we can remove this from the API representation